<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startSimBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopSimBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QSlider" name="moveSpeed">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>行人移动速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>隔离程度</string>
    </property>
   </widget>
   <widget class="QSlider" name="areaDepen">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="isIllcheckBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>感染者</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hasMaskcheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>戴口罩</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showResultBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>展示数据</string>
    </property>
   </widget>
   <widget class="QSlider" name="minComDistance">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>社交距离限制</string>
    </property>
   </widget>
   <widget class="QLabel" name="moveSpeedVal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="comDistanceVal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="areaDepebVal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1277</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
